src/client/inspect.js

```diff
@@ -1,7 +1,15 @@
 import {Inspector} from "observablehq:runtime";
 
-export function inspect(value) {
+export function inspect(value, expand) {
   const node = document.createElement("div");
+  // If the previous inspector was expanded, make the new inspector expand. This
+  // relies on https://github.com/observablehq/inspector/blob/main/src/index.js
```

Comment:
```suggestion
  // relies on https://github.com/observablehq/inspector/blob/63535de10ffb802959324e73553b9825311bfe48/src/index.js#L21-L23
```

Seems a bit hacky to create an empty SPAN element with the appropriate class to trick the inspector into restoring the expanded state. Maybe we should expose the `inspect` method within the inspector so there is more control over how the inspector renders?

Reply:
Ideally as part of https://github.com/observablehq/inspector/issues/12 ?

I'm all for it, but also think this PR has a good Pareto ranking (solving > 90% of the use cases for  < 10% of the work).